src/index.js

```diff
@@ -10,4 +10,10 @@ window.addEventListener('DOMContentLoaded', async () => {
   Utils.inputFocus();
```

Comment:
- [ ] [OPTIONAL] You have done an excellent job with the final touches :100: but I think it could be a good idea to show to submitted score automatically without refreshing the page for a better user experience because currently I have to refresh the page to see the new score and it could be confusing for the users :pushpin: :wink:
![image](https://user-images.githubusercontent.com/81522435/147137236-f42d27d7-de0b-4e5b-b2bc-bf0f3d165dd7.png)
**You can see the OK message but no new scores on the leaderboard**